sql/updates/world/3.3.5/9999_99_99_99_world_burst.sql

```diff
@@ -0,0 +1,9 @@
+UPDATE `spell_script_names` SET `ScriptName`="spell_q13264_q13276_q13288_q13289_burst_at_the_seams" WHERE `spell_id` = 59576;
```

Comment:
```diff
-UPDATE `spell_script_names` SET `ScriptName`="spell_q13264_q13276_q13288_q13289_burst_at_the_seams" WHERE `spell_id` = 59576;
+UPDATE `spell_script_names` SET `ScriptName`='spell_q13264_q13276_q13288_q13289_burst_at_the_seams' WHERE `spell_id` = 59576;
```

Reply:
@Jildor : There is no difference at all between your 2 rows (and the original), what kind of change did you want to suggest?

Reply:
same as above, "x" -> 'x'

Reply:
OK, I see. Reading the TC SQL Guidelines _very strictly_ , there is no mention of double quotation marks (` " " `) in any of the examples on how to submit SQL code. In that sense, the SQL Guidelines only suggest using the single quotation marks like you wrote.

Then again, for those of us who have been focused on SQL fixes submitted to the TDB for some years now, double quotation marks (` " " `) are regularly allowed for text fields to simplify how the text content is inserted. (Your suggestions are more like an academic suggestion for those who want to be very strict about what format is allowed in the SQL code.) :-P  =^.^=